data/gfswind2png.py

```diff
@@ -153,8 +153,11 @@ def slice_image(image, start_y, end_y, start_x, end_x):
 
     tilejson_variables = {}
 
-    tilejson_variables["height"] = 180
-    tilejson_variables["width"] = 360
+    height = 180
```

Comment:
Are these always going to be the same, or should they maybe be defaults in the function call?
```python
def write_image(filename, image, height=180, width=360):
```

Reply:
Well, I think for different dimensions quite a few things would need to change, so I'd leave it as it is for now. I just put them in variables as it made the tiling code a bit easier to understand.